docs/mobile/exercises/README.md

```diff
@@ -3,14 +3,14 @@
 There are roughly 5 exercise types available to you as an author on DataCamp
 Mobile:
 
-| Type                                  | Description                                             |
-| :---                                  | :----------                                             |
-| [Select Output](select-output.md)     | What is the output of this code? Console output         |
-| [Select Table](select-table.md)       | What is the output of this code? Formatted table output |
-| [Select Code](select-code.md)         | Select the code to return the output.                   |
-| [Tap](tap.md)                         | Complete the code to return the output.                 |
-| [Multiple Choice](multiple-choice.md) | Answer the multiple choice question.                    |
-| [Reorder](reorder.md)                 | Reorder the code to return the output.                  |
+| Type                                                    | Description                                             |
+| :---                                                    | :----------                                             |
+| [Select Output](/mobile/exercises/select-output.md)     | What is the output of this code? Console output         |
+| [Select Table](/mobile/exercises/select-table.md)       | What is the output of this code? Formatted table output |
```

Comment:
Maybe mention that tables are mainly / only used for SQL

Reply:
right